src/actors.c

```diff
@@ -1042,7 +1042,10 @@ void func_80299864(Camera *camera, Mat4 arg1, struct Actor *arg2) {
 
     if (func_802B4FF8(arg1, 0) != 0) {
         gDPLoadTLUT_pal256(gDisplayListHead++, D_0D004C68);
-        gSPDisplayList(gDisplayListHead++, D_0600FC70);
+        // Why is a TLUT being used a DL here? That makes no sense
+        // Based on the TLUT being loaded above, this ought to be pointing to another
+        // tree related DL, presumably one not in Moo Moo Farm
+        gSPDisplayList(gDisplayListHead++, d_course_moo_moo_farm_mole_tlut);
```

Comment:
Maybe this isn't mooooo moooo farm but a different course. But I guess that's basically what you just said in that comment.

Reply:
Its what I was trying to say, yeah. could probably reword that to be clearer